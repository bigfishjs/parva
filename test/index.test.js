import React from 'react';
import renderer from 'react-test-renderer';
import Simple from "./fixtures/simple";


test('Simple', async () => {
  
  const component = renderer.create(<Simple />);
  await new Promise(r => setTimeout(r, 100));
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});