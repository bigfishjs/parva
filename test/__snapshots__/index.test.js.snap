// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simple 1`] = `
<div>
  <div>
    1
    list0
  </div>
  <div>
    1
    hello
  </div>
</div>
`;
